internal/cmd/flink/command_statement_list.go

```diff
@@ -14,7 +14,6 @@ func (c *command) newStatementListCommand() *cobra.Command {
 		RunE:  c.statementList,
 	}
 
-	c.addComputePoolFlag(cmd)
```

Comment:
Does this still allow filtering by compute pool, if specified?

Reply:
nope it was never used

Reply:
Can we have a follow-up to add this as a selector, if specified? It should be the same as the UI options

Reply:
done